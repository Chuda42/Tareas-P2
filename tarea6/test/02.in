# Prueba de operaciones de mapping
existeAsociacionEnMap 15
asociarEnMap 12 1.2
asociarEnMap 13 1.3
asociarEnMap 14 1.4
asociarEnMap 15 1.5
existeAsociacionEnMap 15
estaLlenoMap
desasociarEnMap 15
existeAsociacionEnMap 15
asociarEnMap 15 1.5
desasociarEnMap 15
existeAsociacionEnMap 15
asociarEnMap 16 1.6
valorEnMap 12
valorEnMap 13
valorEnMap 14
valorEnMap 16
desasociarEnMap 12
desasociarEnMap 13
desasociarEnMap 14
desasociarEnMap 16
estaLlenoMap
Fin