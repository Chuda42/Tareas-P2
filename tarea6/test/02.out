1>#  Prueba de operaciones de mapping.
2>No existe asociacion.
3>Establecida la asociación.
4>Establecida la asociación.
5>Establecida la asociación.
6>Establecida la asociación.
7>Existe asociacion.
8>No lleno.
9>Eliminada la asociación.
10>No existe asociacion.
11>Establecida la asociación.
12>Eliminada la asociación.
13>No existe asociacion.
14>Establecida la asociación.
15>1.20
16>1.30
17>1.40
18>1.60
19>Eliminada la asociación.
20>Eliminada la asociación.
21>Eliminada la asociación.
22>Eliminada la asociación.
23>No lleno.
24>Fin.
